docker-compose.yml

```diff
@@ -1,8 +1,13 @@
 services:
   frontend:
     build: ./frontend
+    volumes:
+      - ./frontend/:/app
+      - /app/node_modules
```

Comment:
generally ok but if you define in global space (outside services) 
volumes:
 node_modules: 
```

and then use here 
- node_modules:/app/node_modules

that will allows us to cache node modules between  builds of canteiner